CMakeLists.txt

```diff
@@ -344,6 +344,7 @@ if(NOT MSVC)
   set (CMAKE_LINKER_FLAGS_DEBUG "${CMAKE_STATIC_LINKER_FLAGS_DEBUG} -fno-omit-frame-pointer -O0")
   set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fno-math-errno")
   set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fno-trapping-math")
+  set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fcolor-diagnostics")
```

Comment:
shouldn't it be `fdiagnostics-color`?

Reply:
"GCC has supported an option for this since GCC 4.9, called -fdiagnostics-color. On Clang, there’s an equivalent option with a frustratingly similar name: -fcolor-diagnostics."

fml